<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>668</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Academy Engraved LET</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(121, 121, 121);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_60">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_70">
      <item>
       <widget class="QPushButton" name="face_detection_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Academy Engraved LET</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Face Detection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="face_recognition_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Academy Engraved LET</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Face Recognition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ROC_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Academy Engraved LET</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ROC</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>200</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_69">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_61">
          <item>
           <widget class="QLabel" name="original_image">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="face_detection_image">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="detection_submit_btn">
          <property name="font">
           <font>
            <family>Academy Engraved LET</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Detect Faces</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>911</width>
          <height>591</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="face_recognition_image">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="reference">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>300</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_62" stretch="1,1,1">
           <item>
            <widget class="QPushButton" name="pca_btn">
             <property name="font">
              <font>
               <family>Academy Engraved LET</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Recognize</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="id_label">
             <property name="styleSheet">
              <string notr="true">font: 700 18pt &quot;Segoe UI&quot;;</string>
             </property>
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_72" stretch="0,1,4,1,2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Academy Engraved LET</family>
            <pointsize>16</pointsize>
            <underline>false</underline>
           </font>
          </property>
          <property name="text">
           <string>Choose Threshold value:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSlider" name="threshold_slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="show_roc_button">
          <property name="font">
           <font>
            <family>Academy Engraved LET</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Show ROC Curve</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1043</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpload_Image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpload_Image">
   <property name="text">
    <string>Upload Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>threshold_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>981</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
